File: Java/olamundo/src/main/java/br/com/iteris/universidade/olamundo/domain/dto/AvaliacaoResponse.java
package br.com.iteris.universidade.olamundo.domain.dto;

import lombok.AllArgsConstructor;
import lombok.Data;

@Data
@AllArgsConstructor
public class AvaliacaoResponse {
    private int idAvaliacao;
    private int idAlbum;
    private int nota;
    private String comentario;
}

File: C#/PrimeiraWebAPI/PrimeiraWebAPI/Domain/DTO/AvaliacaoResponse.cs
using PrimeiraWebAPI.Domain.Entity;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
namespace PrimeiraWebAPI.Domain.DTO
{
	public class AvaliacaoResponse
	{
		public AvaliacaoResponse(Avaliacao avaliacao)
		{
			IdAvaliacao = avaliacao.IdAvaliacao;
			IdAlbum = avaliacao.IdAlbum;
			Nota = avaliacao.Nota;
			Comentario = avaliacao.Comentario;
		}
		public int IdAvaliacao { get; set; }
		public int IdAlbum { get; set; }
		public int Nota { get; set; }
		public string Comentario { get; set; }
	}
}